example/config/vm.args

```diff
@@ -0,0 +1,2 @@
+-name example@127.0.0.1
+-setcookie bogus
```

Comment:
Without this we'd get issues in CI. Actually, I don't think we're testing the example in CI and I'm gonna do it now.

Reply:
458ea3c